libs/external-router/src/router/helper/insert-route-before-wildcard.ts

```diff
@@ -4,12 +4,13 @@ import {
 } from '@angular/router';
 
 import { DaffExternalRouterNoWildcardError } from '../../errors/no-wildcard';
+import { DaffExternalRouterRouteInserter } from '../../model/route-inserter.type';
 
 /**
  * Inserts a Route into Routes right before the wildcard route.
  * If no wildcard is found, it fails fast with a `DaffExternalRouterNoWildcardError`.
  */
-export const insertRouteBeforeWildCard = (
+export const daffExternalRouterInsertRouteBeforeWildCard: DaffExternalRouterRouteInserter = (
```

Comment:
This function name is wild. Since this type of function won't exist elsewhere, I think we can simply remove the package prefix.

Reply:
its possible that the app dev might want to delegate to this function so I think its best to export it.

Reply:
I agree, I'm simply arguing to reduce the length of the function name.

Reply:
`daffExternalRouterRootInsertionStrategy`? I think the type names should also be changed to "insertion strategy", as that is more descriptive.

Reply:
daffInsertRoutrBeforeWildCardStrategy

Reply:
changed